src/editors/playgroundController.ts

```diff
@@ -140,26 +147,45 @@ export default class PlaygroundController {
         );
 
         if (confirmRunAll !== 'Yes') {
-          return Promise.resolve(false);
+          return resolve(false);
         }
       }
 
       const activeEditor = vscode.window.activeTextEditor;
       const codeToEvaluate = activeEditor?.document.getText() || '';
       let result;
 
-      try {
+      // Show a running progress in the notification area with support for cancellation
+      await vscode.window.withProgress({
+        location: ProgressLocation.Notification,
+        title: 'Running a playground...',
```

Comment:
Maybe `Running MongoDB playground...` maybe we could also put the name of the database it's running against?